include/sisl/fds/buffer.hpp

```diff
@@ -41,14 +42,14 @@ struct blob {
 
 struct sg_list {
     uint64_t size; // total size of data pointed by iovs;
-    std::vector< iovec > iovs;
+    folly::small_vector< iovec > iovs;
```

Comment:
Why are we switching to `folly::small_vector` here; and if it does make sense is using a `MaxInline=1` the optimal?

Reply:
good catch, it should be 4;
The reason for changing to small_vector is we believe most of the cases the iovs will not be greater than 4, based on the max IO size we have in our system;